src/mgmt/srv_internal.h

```diff
@@ -63,11 +63,9 @@ void ds_mgmt_svc_put(struct mgmt_svc *svc);
 int ds_mgmt_group_update_handler(struct mgmt_grp_up_in *in);
 
 /** srv_pool.c */
-int ds_mgmt_create_pool(uuid_t pool_uuid, const char *group, char *tgt_dev,
-			d_rank_list_t *targets, size_t scm_size,
-			size_t nvme_size, daos_prop_t *prop, uint32_t svc_nr,
-			d_rank_list_t **svcp, int domains_nr,
-			uint32_t *domains);
+int ds_mgmt_create_pool(uuid_t pool_uuid, const char *group, char *tgt_dev, d_rank_list_t *targets,
```

Comment:
nit: We should generally avoid making style-only changes, especially in files that are otherwise untouched by the PR.